views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -838,18 +851,18 @@ define([
              * Update the reset recording button with the number of remaining attempts
              */
             updateResetCount: function updateResetCount() {
-                var remaining = this.config.maxRecords - this._recordsAttempts - 1,
+                var remaining = this.config.maxRecords - this._recordsAttempts,
```

Comment:
I don't think this has worked, because the interaction which previously displayed "2 > 1 > 0" now goes "3 > 2 > 1" so the final state is a disabled trashcan with "(1)" after it.
<img width="328" height="89" alt="Screenshot 2025-08-04 at 09 18 55" src="https://github.com/user-attachments/assets/569b987c-9b91-4690-980a-5855ce01cf13" />

If possible, let's stick with the old numbers, but just make sure we restore the displayed value, which isn't happening now.

Reply:
It still has an issue for me after https://github.com/oat-sa/extension-tao-itemqti-pci/pull/445/commits/f51954de848a9227e985757451d7a241df256150.

I have an interaction with `"maxRecords": "3"`. I go like this:
(Shows 2 deletes) Record, stop, delete (now shows 1). Record, stop, delete (now shows 0). Record, stop (now shows 0 and disabled).
Navigate away and back. The trashcan shows -1:

<img width="356" height="102" alt="Screenshot 2025-08-07 at 14 52 20" src="https://github.com/user-attachments/assets/cc4b8d72-ef34-4941-b4d8-63e3162f7510" />